### ISSUE TITLE
Filter list for uBlacklist

### ISSUE THREAD

**Issue by dooort729** [WindowsAurora/FMHYFilterlist#13]
This is my addition; I highlighted the real sites mentioned in this list.
```
# Real sites highlighted
@1*://thepiratebay.org/*
@1*://fitgirl-repacks.site/*
@1*://1337x.to/*
@1*://singlelogin.rs/*
@1*://singlelogin.re/*
```

The only change made is to adapt it to the format of uBlacklist.
```
# Unsafe game sites
*://*.igg-games.com/*
*://*.igg-games.cc/*
*://*.bluemediafile.online/*
*://*.bluemediafile.sbs/*
*://*.bluemediafile.site/*
*://*.bluemediafiles.com/*
*://*.bluemediafiles.eu/*
*://*.bluemediafiles.homes/*
*://*.bluemediafiles.xyz/*
*://*.bluemediastorage.online/*
*://*.bluemediaurls.lol/*
*://*.oceanofgames.com/*
*://*.oceansofgamess.com/*
*://*.oceantogames.com/*
*://*.oceanofgamesu.com/*
*://*.oceanofapks.com/*
*://*.oceanofgame.cc/*
*://*.blackboxrepacke.com/*
*://*.crackingpatching.com/*
*://*.www.crackingpatching.org/*
*://*.crackingpatching.org/*
*://*.cracked-games.org/*
*://*.www.wifi4games.com/*
*://*.wifi4games.com/*
*://*.wifi4games.net/*
*://*.gamefabrique.com/*
*://*.gamefabrique.ru/*
*://*.steamunlocked.net/*

# Pirate bay proxies And clones
*://*.pirate-bays.net/*
*://*.piratebayproxy.net/*
*://*.www.pirateproxy-bay.com/*
*://*.pirateproxy-bay.com/*
*://*.thepiratebay3.co/*
*://*.thepiratebay10.info/*
*://*.thepiratebay7.com/*
*://*.thepiratebay0.org/*
*://*.thepiratebay10.org/*
*://*.pirateproxy.live/*
*://*.thehiddenbay.com/*
*://*.piratebay.live/*
*://*.thepiratebay.zone/*
*://*.tpb.party/*
*://*.thepiratebay.party/*
*://*.piratebay.party/*
*://*.piratebayproxy.live/*
*://*.pirateproxylive.org/*
*://*.thepiratebay10.xyz/*
*://*.thepiratebay1.live/*
*://*.thepiratebays.info/*
*://*.thepiratebays.live/*
*://*.thepiratebay1.top/*
*://*.thepiratebay1.info/*
*://*.thepiratebay.rocks/*
*://*.thepiratebay.vet/*

# Popup ads on Pirate bay
*://*.wpadmngr.com/*
*://*.iconcardinal.com/*

# Fake Fitgirl repacks sites
*://*.fitgirl-repack.site/*
*://*.fitgirl-repack.com/*
*://*.fitgirlrepacks.org/*
*://*.fitgirl-repacks.website/*
*://*.fitgirlrepacks.co/*
*://*.fitgirlrepack.games/*
*://*.fitgirl.cc/*
*://*.fitgirlrepacksite.com/*
*://*.fitgirlrepacks.in/*
*://*.fitgirl-repacks.cc/*
*://*.fitgirl-repacks.to/*
*://*.ww9.fitgirl-repacks.xyz/*
*://*.fitgirl-repacks.xyz/*
*://*.fitgirl-repack.net/*
*://*.fitgirlpack.site/*
*://*.fitgirl-repack.org/*
*://*.fitgirlrepacks.pro/*
*://*.fitgirlrepack.games/*
*://*.fitgirltorrent.com/*
*://*.fitgirl-repacks.vip/*
*://*.fitgirlrepackz.com/*
*://*.fitgirl-repacks.theproxy.vip/*
*://*.fitgirl-repacks.us/*
*://*.fitgirl-repacks.unb/*
*://*.fitgirl-repacks.in/*

# Software / App Sites
*://*.getintopc.com/*
*://*.getintopc.cc/*
*://*.igetintopc.com/*
*://*.www.getintomac.net/*
*://*.getintomac.net/*
*://*.getinmac.com/*
*://*.getitintopc.com/*
*://*.www.sadeempc.com/*
*://*.sadeempc.com/*
*://*.sadeemapk.com/*
*://*.karanpc.com/*
*://*.filecr.com/*
*://*.ftuapps.io/*
*://*.ftuapps.dev/*
*://*.www.ftuapps.me/*
*://*.ftuapps.me/*
*://*.s0ft4pc.com/*
*://*.soft4pc.net/*
*://*.portable4pc.com/*
*://*.crackshash.org/*
*://*.crackshash.com.atlaq.com/*
*://*.haxnode.net/*
*://*.mrepo.org/*
*://*.mainrepo.org/*
*://*.yasdl.com/*
*://*.allpcworld.com/*
*://*.allpcworld.co/*
*://*.allpcworlds.com/*
*://*.rsload.net/*
*://*.store.app-valley.vip/*
*://*.app-valley.vip/*
*://*.appvalley-apk.com/*
*://*.tutubox.io/*
*://*.www.cnet.com/*
*://*.cnet.com/*
*://*.download.com/*
*://*.www.zdnet.com/*
*://*.zdnet.com/*
*://*.en.softonic.com/*
*://*.softonic.com/*
*://*.ezcrack.info/*

# Torrent Sites / Clients
*://*.thekickasstorrents.com/*
*://*.kickass-official.blue/*
*://*.kickasstorrents.to/*
*://*.vstorrent.org/*
*://*.www.utorrent.com/*
*://*.utorrent.com/*
*://*.www.bittorrent.com/*
*://*.bittorrent.com/*
*://*.www.bitcomet.com/*
*://*.bitcomet.com/*
*://*.www.frostwire.com/*
*://*.frostwire.com/*
*://*.www.bitlord.com/*
*://*.bitlord.com/*
*://*.bitlorddl.com/*
*://*.1377x.to/*

# Software / Apps
*://*.limewire.com/*
*://*.downloadlynet.ir/*
*://*.downloadly.ir/*
*://*.downloadly.net/*
*://*.downloadlyir.com/*
*://*.www.opera.com/*
*://*.opera.com/*
*://*.www.mcafee.com/*
*://*.mcafee.com/*
*://*.www.mcafeemobilesecurity.com/*
*://*.mcafeemobilesecurity.com/*
*://*.avast.com/*
*://*.www.avg.com/*
*://*.avg.com/*
*://*.www.gendigital.com/*
*://*.gendigital.com/*
*://*.us.norton.com/*
*://*.norton.com/*
*://*.login.norton.com/*
*://*.www.nortonlifelock.com/*
*://*.nortonlifelock.com/*
*://*.www.avira.com/*
*://*.avira.com/*
*://*.my.avira.com/*
*://*.www.ccleaner.com/*
*://*.ccleaner.com/*
*://*.www.ccleanercloud.com/*
*://*.ccleanercloud.com/*
*://*.www.privateinternetaccess.com/*
*://*.privateinternetaccess.com/*
*://*.www.expressvpn.com/*
*://*.expressvpn.com/*
*://*.zenmate.com/*
*://*.www.cyberghostvpn.com/*
*://*.cyberghostvpn.com/*
*://*.www.kape.com/*
*://*.kape.com/*
*://*.www.vpnmentor.com/*
*://*.vpnmentor.com/*
*://*.www.wizcase.com/*
*://*.wizcase.com/*
*://*.bluesmods.com/*
*://*.bluekik.pro/*
*://*.kik.com/*
*://*.tlauncher.org/*
*://*.tlauncher.co.com/*
*://*.polymc.org/*
*://*.www.gshade.org/*
*://*.gshade.org/*
*://*.www.totalav.com/*
*://*.totalav.com/*
*://*.help.totalav.com/*
*://*.www.pcprotect.com/*
*://*.pcprotect.com/*

# Fake Zlib Sites
*://*.zlib.is/*
*://*.z-lib.ai/*
*://*.z-lib.club/*
*://*.z-lib.dev/*
*://*.z-lib.ink/*
*://*.z-lib.li/*
*://*.z-lib.nl/*
*://*.zlib.to/*
*://*.zlibrary.to/*
*://*.zlibrary.is/*
*://*.z-lib.io/*
*://*.z-lib.ee/*
*://*.z-lib.tf/*
*://*.z-lib.wf/*
*://*.z-lib.yt/*
*://*.z-lib.id/*
*://*.z-lib.ws/*
*://*.z-lib.lol/*
*://*.zlibis.com/*
*://*.zlibrary-global.com/*
*://*.zlib-official.com/*
*://*.zlibrary.lol/*
*://*.z-library.live/*
*://*.z-library.wiki/*
*://*.z-lib.wiki/*
*://*.z-library.blog/*
*://*.z-library.help/*
*://*.z-library-proxy.com/*

# Fake Windows Activators
*://*.activator-kmspico.net/*
*://*.ativadorkmspico.com/*
*://*.dazactivator.com/*
*://*.get-kmspico.com/*
*://*.getkmspico.com/*
*://*.kms-pc.com/*
*://*.kms-tool.com/*
*://*.kmsautoplus.su/*
*://*.kmsmatrix.com/*
*://*.kmspico-activator-crack.com/*
*://*.kmspico-activator-free.com/*
*://*.kmspico-activator.in/*
*://*.kmspico-mafia.com/*
*://*.kmspico-official.xyz/*
*://*.kmspico-oficial.com/*
*://*.kmspico-tools2.site/*
*://*.kmspico.co/*
*://*.kmspico.icu/*
*://*.kmspico.io/*
*://*.kmspico.space/*
*://*.kmspico.uno/*
*://*.kmspico.ws/*
*://*.kmspico10.com/*
*://*.kmspicoactivator.net/*
*://*.kmspicoativador.org/*
*://*.kmspicodownloads.com/*
*://*.kmspicofficial.com/*
*://*.kmspicoofficial.com/*
*://*.kmspicoportable.com/*
*://*.massgrav.shop/*
*://*.massgrav.zapto.org/*
*://*.massgrave.hopto.org/*
*://*.massgraves.online/*
*://*.mrkmspico.com/*
*://*.official-kmspico.co/*
*://*.official-kmspico.com/*
*://*.official-kmspico.org/*
*://*.officialkmspico.com/*
*://*.officialkmspico.download/*
*://*.officialkmspico.net/*
*://*.officials-kmspico.com/*
*://*.oficial-kmspico.com/*
```

**Comment by dooort729** [comment]
Having a filter list for uBlacklist would be great.

**Comment by MIOGMIOG** [comment]
Thanks for the suggestion, I will add support for this in the next commit (which should be released soon)

**Comment by dooort729** [comment]
Great, but isn't highlighting a good thing too?

**Comment by MIOGMIOG** [comment]
> Great, but isn't highlighting a good thing too?

Well, the current system for generating blocklist files is not very good, but I guess it could be implemented (with much more features I have in mind), but I really do not want to implement it right now unfortunately, also I want to make the current blocklists somewhat universal right now
TLDR: It might get added in the future as a different filterlist "format".
